XTS/bridge-jakarta/pom.xml

```diff
@@ -0,0 +1,224 @@
+                  </manifest>
+                </jar>
+              </target>
+            </configuration>
+          </execution>
+          <execution>
+            <id>transform-sources-jar</id>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <target>
+                <property name="plugin_classpath" refid="maven.plugin.classpath"></property>
+                <ac:propertyregex xmlns:ac="antlib:net.sf.antcontrib" global="true" input="${org.jboss.narayana.xts:bridge:jar}" property="source" regexp="\.jar$" replace="-sources.jar"></ac:propertyregex>
+                <java classname="org.eclipse.transformer.jakarta.JakartaTransformer">
+                  <arg value="${source}"></arg>
+                  <arg value="${project.build.directory}/${project.build.finalName}-sources.jar"></arg>
+                  <arg value="-q"></arg>
+                  <arg value="-o"></arg>
+                  <arg value="-tr"></arg>
+                  <arg value="${project.basedir}/../../jakarta-rules/jakarta-renames.properties"></arg>
+                  <classpath>
+                    <pathelement path="${plugin_classpath}"></pathelement>
+                  </classpath>
+                </java>
+              </target>
+            </configuration>
+          </execution>
+          <execution>
+            <id>transform-javadoc</id>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <target>
+                <property name="plugin_classpath" refid="maven.plugin.classpath"></property>
+                <ac:propertyregex xmlns:ac="antlib:net.sf.antcontrib" global="true" input="${org.jboss.narayana.xts:bridge:jar}" property="source" regexp="\.jar$" replace="-javadoc.jar"></ac:propertyregex>
+                <java classname="org.eclipse.transformer.jakarta.JakartaTransformer">
+                  <arg value="${source}"></arg>
+                  <arg value="${project.build.directory}/${project.build.finalName}-javadoc.jar"></arg>
+                  <arg value="-q"></arg>
+                  <arg value="-o"></arg>
+                  <arg value="-tr"></arg>
+                  <arg value="${project.basedir}/../../jakarta-rules/jakarta-renames.properties"></arg>
+                  <classpath>
+                    <pathelement path="${plugin_classpath}"></pathelement>
+                  </classpath>
+                </java>
+              </target>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <inherited>false</inherited>
+          </plugin>
```

Comment:
Just a note: I like when the `nexus-staging` plugin is used in the `release` profile [personal preference :-) ]